saleor/account/models.py

```diff
@@ -162,6 +164,26 @@ class Meta:
             (AccountPermissions.MANAGE_STAFF.codename, "Manage staff."),
         )
 
+    def __init__(self, *args, **kwargs):
+        super(User, self).__init__(*args, **kwargs)
```

Comment:
A detail, but the new-style of writing this is `super().__init__(*args, **kwargs)`.